contributing/developer-blog.md

```diff
@@ -0,0 +1,66 @@
+
+* [Contributing](#contributing)
+* [Core Principles](#core-principles)
+
+## Contributing
+
+The dbt Developer Blog is a place where analytics practitioners can go to share their knowledge with the community. Analytics Engineering is a discipline we’re all  building together. The developer blog exists to cultivate the collective knowledge that exists on how to build and scale effective data teams.
+
+### What makes a good developer blog post?
+
+- The short answer: Practical, hands on analytics engineering tutorials and stories
+    - [Slim CI/CD with Bitbucket](https://docs.getdbt.com/blog/slim-ci-cd-with-bitbucket-pipelines)
+    - [So You Want to Build a dbt Package](https://docs.getdbt.com/blog/so-you-want-to-build-a-package)
+    - [Founding an Analytics Engineering Team](https://docs.getdbt.com/blog/founding-an-analytics-engineering-team-smartsheet)
+- See the Developer Blog core principles
+
+### How do I submit a proposed post?
+
+To submit a proposed post, open a `Contribute to the dbt Developer Blog` issue on the [Developer Hub repo](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose). You will be asked for:
+
+- A short (one paragraph) summary of the post you’d like to publish
+- An outline of the post (example)
```

Comment:
We should make this a link to an example before we publish or as part of this PR?

Reply:
Ahh yes I was looking for a good example and couldn't find one. May drop the example for now and we can add in when we have a good example outline.